mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/tilsagn/db/TilsagnQueries.kt

```diff
@@ -90,6 +94,22 @@ class TilsagnQueries(private val session: Session) {
         )
     }
 
+    fun updateGjenstaendeBelop(id: UUID, belop: Int) = with(session) {
```

Comment:
Navnet  tilsier at `belop` er nytt beløp, men query'et har en annen logikk. Foreslår at vi kaller denne for `setGjenstaendeBelop` og gjør logikken i service

Reply:
Hadde også vært nice med en test på dette (i utbetaling-service?)

Reply:
Ja gir mening :)